R/report.R

```diff
@@ -70,12 +70,15 @@ report <- function(
   csl <- system.file("research-institute-for-nature-and-forest.csl",
                      package = "INBOmd")
 
+  style <- ifelse(style == "Flanders" & lang == "fr", "Flandre", style)
```

Comment:
we need to enforce `style == "Flanders"` for all languages except Dutch.

Reply:
Why?  A French version of the website `www.vlaanderen.be` exists, and it seems more appropriate (and user friendly) to link to this website in the french version of our reports.  Maybe copying the whole `.sty` file to only change this is not the best option, but I don't know how to parameterise the language in a `.sty` file (and I noticed for Dutch a separate file was made as well).